bugbug/model.py

```diff
@@ -170,7 +170,7 @@ def train(self, importance_cutoff=0.15):
         X_iter, y_iter = split_tuple_iterator(self.items_gen(classes))
 
         # Extract features from the items.
-        X = self.extraction_pipeline.fit_transform(X_iter)
+        X = self.extraction_pipeline.fit_transform([item for item in X_iter])
```

Comment:
We can probably simplify a few things if we do this, but let's do it later as a follow-up.